.github/workflows/test.yml

```diff
@@ -25,16 +25,16 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        ckan-version: ["2.10", 2.9, 2.9-py2, 2.8, 2.7]
+        ckan-version: ["2.10", 2.9]
```

Comment:
can you add 2.11 and mark it has 'ignore if failed', might as well show that its not ready in cicd testing then to not have it in here.
i.e.
```
    strategy:
      matrix:
        ckan-version: ["2.11", "2.10", "2.9"]
        include:
          - version: "2.11"
            continue-on-error: true  # Allow failure on version 2.11 so work further development work can continue
```